src/book/02-system/04-other/01-nubook.mdx

```diff
@@ -167,9 +169,7 @@ The following example shows four images in a 2x2 grid with a caption:
 
 The [columns](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns) and [rows](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns) are specified using CSS grid syntax. `fr` is a fractional unit that specifies a fraction of the total available space (width for columns, and height for rows). `columns='1fr 1fr'` creates two columns of equal width, while `rows='1fr 1fr'` creates two rows of equal height.
 
-See the [original pull request](https://github.com/NUbots/NUbook/pull/20) for more examples.
```

Comment:
Any reason for taking the links to the examples out? Same with the tabbed images and kitchen sink links below.

Reply:
Nevermind, saw the link above.